utils/remote_control/navigator_emergency_control/nodes/nodes/navigator_emergency.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+
+from remote_control_lib import RemoteControl
+import rospy
+from sensor_msgs.msg import Joy
+
+
+__maintainer__ = "Anthony Olive"
```

Comment:
Ah, I was wondering what I should do about that. I only changed one line of code so I thought Anthony should be kept as author. But I also thought if something goes wrong with the node, my name should be there so Anthony can have deniability. I'll go with your suggestion if that's standard.